tests/complete_item_spec.lua

```diff
@@ -9,9 +9,8 @@ describe("NotesCompleteItem", function()
     helpers.setup_test_env()
     helpers.clear_buffer()
```

Comment:
is `clear_buffer` used anywhere else? if not, maybe it can be a part of `setup_test_env`